index.html

```diff
@@ -2139,8 +2190,203 @@ <h2>Sessions</h2>
  an <a>intermediary node</a> will also <a>close the session</a> of
  the <a>associated session</a>.
 
-<p>All <a>commands</a>, except <a>New Session</a> and <a>Status</a>,
- have an associated <dfn>current session</dfn>,
+<p>A <a>remote end</a> has an associated
+ <dfn>maximum timeout for async session creation</dfn> in milliseconds
+ (an Integer). It may be <a><code>null</code></a> if “unlimited”.
+
+ <p>A <a>remote end</a> has an associated
+  <dfn>wait time for async session creation</dfn> in milliseconds
+  (an Integer). It may be <a><code>null</code></a> if “unlimited”.
+
+<p>A <dfn>session creation job</dfn> is a task to create a session:</a>
+
+<table class=simple>
+<tr>
+<th>Field
+<th>Default
+<th>JSON key
+<th>Optional<sup>†</sup>
+<th>Nullable
+<th>Description<sup>†</sup>
+</tr>
+
+<tr>
+<td><dfn>Session creation job ID</dfn>
+<td>
+<td>"<code>sessionCreationJobId</code>"
+<td>
+<td>
+<td>A string representation of a <a>UUID</a> used to uniquely identify this job.
+</tr>
+
+<tr>
+<td><dfn>Session creation status</dfn>
+<td>"<code>queuing</code>"
+<td>"<code>status</code>"
+<td>
+<td>
+<td>String indicating session creation status,
+ one of "<code>queueing</code>", "<code>creating</code>", "<code>created</code>",
+ "<code>failed</code>", and "<code>cancelled</code>".
+</tr>
+
+<tr>
+<td><dfn>Capabilities request</dfn>
+<td>Empty JSON <a>Object</a>
+<td>"<code>capabilitiesRequest</code>"
+<td>
+<td>
+<td>The capabilities request.
+</tr>
+
+<tr>
+<td><dfn>Session creation timeout</dfn>
+<td>600,000
+<td>"<code>timeout</code>"
+<td>
+<td>✓
+<td>Specified as an <a>integer</a> in milliseconds.
+ If session creation is not achieved within this timeout,
+ the job should be failed.
+ If <a><code>null</code></a>,
+ there is no timeout beyond implementation limits.
+</tr>
+
+<tr>
+<td><dfn>Session creation requested time</dfn>
+<td>
+<td>"<code>requestedTime</code>"
+<td>
+<td>
+<td>When the session creation was requested as a <a>millisecond timestamp</a>.
+</tr>
+
+<tr>
+<td><dfn>Session creation ended time</dfn>
+<td>
+<td>"<code>endedTime</code>"
+<td>✓
+<td>
+<td>When the job ended in success or failure as a <a>millisecond timestamp</a>.
+</tr>
+
+<tr>
+<td><a>Session ID</a>
+<td>
+<td>"<code>sessionId</code>"
+<td>✓
+<td>
+<td><a>Session ID</a> of the created session.
+</tr>
+
+<tr>
+<td><dfn>Session creation events</dfn>
+<td>Empty JSON <a>List</a>
+<td>"<code>events</code>"
+<td>
+<td>
+<td>Time-ordered list of session creation events.
+</tr>
+
+<tr>
+<td><dfn>Session creation data</dfn>
+<td></a>
+<td>"<code>data</code>"
+<td>✓
+<td>
+<td>Object containing implementation-defined data associated with this job.
```

Comment:
We should add a not that this will be not further specified as it is implementation specific.